### PR TITLE
feat: add react-admin monorepo

### PR DESCRIPTION
**Pull request by herndlm**
## Changes:

Adds a group for the react-admin monorepo packages from https://github.com/marmelab/react-admin/tree/v3.17.2/packages

## Context:

If react-admin creates a new release, all or at least multiple packages are tagged and I often get multiple PRs from renovate. Most of the time I just manually adapt the first one that updates react-admin to include all the ra-* updates too. Otherwise there might actually be dependency problems occurring. To ease that process I thought it makes sense to have default grouping active for that. 

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please tick one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [x] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by viceice — CHANGES_REQUESTED** [review]
Please add a monorepo config to 
https://github.com/renovatebot/renovate/blob/a22eacf58db302a3e8d379c44d140870cc4fb1f2/lib/config/presets/internal/monorepo.ts

**Comment by herndlm** [comment]
adapted and did a successful local test, thx for the fast review!
image

File: lib/config/presets/internal/group.ts [hunk 1]
```diff
@@ -583,6 +583,17 @@ const staticGroups = {
...
+    description: 'Group react-admin packages together',
+    packageRules: [
+      {
+        matchPackageNames: ['react-admin'],
+        matchPackagePatterns: ['^ra-'],
+        groupName: 'react-admin packages',
+        groupSlug: 'react-admin',
+      },
+    ],
+  },
```
Comment (viceice) [inline lib/config/presets/internal/group.ts:596]:
```suggestion
```

forgot to revert this?

Comment (herndlm) [reply lib/config/presets/internal/group.ts:596]: Oooh, now I got how this works. Sorry, misunderstood it..
Adapted and re-tested, still working as expected

**Review by viceice — APPROVED** [review]

**Comment by herndlm** [comment]
Strange, shouldn't those commits have been squashed on main? @rarkins

**Comment by rarkins** [comment]
Yes, but I'd temporarily enabled rebase merging for the v26 branch and forgot to reset it :(

Thankfully your commits were clean enough :)

**Pull request merged**